karton/dashboard/templates/index.html

```diff
@@ -19,7 +19,14 @@ <h3 class="text-center">binds</h3>
         <td>
           <a href="queue/{{ queue_name }}">{{ queue_name }}</a>
           <div>
-            <span class="badge badge-info">{{ queue.bind.version }}</span>
+            <span class="badge badge-info">
+              📦{{ queue.bind.version }}
```

Comment:
no `v` here but there is one below. Looks inconsistent :thinking:

Reply:
I've thought that we either use :package: or `v`, but it sill looks okay :+1: